R/ff_prep_refactored.R

```diff
@@ -144,8 +147,8 @@ quality_check <- function(dates, country, shape, tiles, datafolder, shrink) {
     stop("No dates were given")
   }
 
-  if (as.Date(min(dates)) < as.Date("2021-01-01")) {
-    stop("The earliest date available is 2021-01-01")
+  if (as.Date(min(dates)) < as.Date(config$EARLIEST_DATA_DATE)) {
+    stop(paste0("The earliest date available is ", config$EARLIEST_DATA_DATE))
```

Comment:
I this there is typo here, paste0?

Reply:
paste0 is a R function to concat string without adding whitespace.